### PR TITLE
Add localization for hardcoded text strings

### PR DESCRIPTION
**Pull request by EEUAAAIA**
I searched all the lines of harcoded code and I defined keys in the fr.yml file then I used the t() method to translate these keys into the corresponding views.

### CODE REVIEW

File: app/views/default/pages/homepage.html.erb [hunk 1]
```diff
@@ -106,7 +106,7 @@
   <section class="journal-section">
     <article class="container" role="article">
       <h2 data-aos="fade-right" class="title title--xxl journal-section__title">
-        My journal
+        <%= t('my_journal')%>
```
Comment (bolom) [inline app/views/default/pages/homepage.html.erb:109]: Be careful and ensure to check if it doesn't break the design.

File: app/views/default/pages/journal_items.html.erb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
     <article class="container container--lg">
-      <a href="/">Home page</a>
+      <a href="/"><%= t('home_page')%></a>
     </article>
   </div>
   <section class="section">
     <article class="container container--text" role="article">
       <div class="journal">
-        <h1><%= current_page.title %></h1>
+        <h1><%= t('my_journal') %></h1>
```
Comment (bolom) [inline app/views/default/pages/journal_items.html.erb:10]: No, here the data is from the variable current_page. The value is not hardcoded.

**Review by bolom — CHANGES_REQUESTED** [review]

File: app/views/default/pages/homepage.html.erb (hunk 1 above)
Comment (EEUAAAIA) [reply app/views/default/pages/homepage.html.erb:109]: I had nothing on my locale but it's done

File: app/views/default/pages/journal_items.html.erb (hunk 2 above)
Comment (EEUAAAIA) [reply app/views/default/pages/journal_items.html.erb]: Done

File: app/views/default/pages/homepage.html.erb (hunk 1 above)
Comment (EEUAAAIA) [reply app/views/default/pages/homepage.html.erb:109]: https://github.com/bolom/gilbertmarcel/blob/d053d178f746cf4292ec7fb44fbfac8843eee021/app/views/default/pages/homepage.html.erb#L108C7-L111C12

**Pull request merged**